app/auto_responder.go

```diff
@@ -43,11 +43,21 @@ func (a *App) SendAutoResponse(channel *model.Channel, receiver *model.User) (bo
 		return false, nil
 	}
 
+	rootID := post.Id
```

Comment:
Is this really needed? Can we rely on `a.CreatePost` doing the "right thing" can just use `post.RootId` in any case?

Reply:
Not sure what do you mean. I guess you are saying that we should only define rootID and not define parentID since CreatePost will make them the same either way?

Reply:
Can we just pass `post.RootId` as `rootID` and remove the `if` check?

Reply:
No, since RootId will be empty for non threaded messages.